docs/plans/2026-03-12-001-docs-complete-documentation-restructure-plan.md

```
@@ -0,0 +1,330 @@
+- `OrderPlaced` event — published when an order is created
+- `ProcessPayment` command — sent to payment service
+- `PaymentCompleted` event — published after successful payment
+- `ShipOrder` command — sent to shipping service
+- `OrderShipped` event — published after shipment
+- `SendOrderConfirmation` command — notification to customer
+
+This single domain covers: events, commands, multiple handlers, outbox (order creation + event in same TX), inbox (payment deduplication), routing (different channels for different bounded contexts).
+
+### Architecture
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)